installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc

```diff
@@ -2,6 +2,9 @@
 = Configuration files
```

Comment:
We need to update the _topic_map.yml to include this Configuration files content. When I do a local build of your branch, the content is not showing up.

Reply:
Currently, the new content is building fine in installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc, however. Is the placement intended?

Reply:
Yes. The engineers wanted a sub-heading, because all those modules are related. So we did a sub-assembly.